code/modules/mob/living/carbon/xenomorph/castes/defender/castedatum_defender.dm

```diff
@@ -27,7 +27,10 @@
 	evolution_threshold = 80
 	upgrade_threshold = TIER_ONE_YOUNG_THRESHOLD
 
-	evolves_to = list(/mob/living/carbon/xenomorph/warrior)
+	evolves_to = list(
+	/mob/living/carbon/xenomorph/warrior,
+	/mob/living/carbon/xenomorph/bull
```

Comment:
need comma to not die from liner

/mob/living/carbon/xenomorph/bull,

Reply:
guh